class.jetpack-signature.php

```diff
@@ -204,6 +204,17 @@ function sign_request( $token = '', $timestamp = 0, $nonce = '', $body_hash = ''
 		);
 
 		$normalized_request_pieces = array_merge( $normalized_request_pieces, $this->normalized_query_parameters( isset( $parsed['query'] ) ? $parsed['query'] : '' ) );
+		$flat_normalized_request_pieces = array();
+		foreach ($normalized_request_pieces as $piece) {
+			if ( is_array( $piece ) ) {
+				foreach ($piece as $subpiece) {
```

Comment:
Spaces in parens missing (same two lines above).

Reply:
🤦‍♂️  fixed in a11625a5dc0489bd62f9bca006d0967810fdd372